manuscript/10-SoftQual.txt

```diff
@@ -559,11 +559,11 @@ Off-the-Shelf Software) (COTS) και παροχή υπηρεσιών μετά 
 Μια τεχνική επιθεώρηση μπορεί να απαιτεί οι υποχρεωτικές εισροές να
 είναι στη θέση τους προκειμένου να προχωρήσει σε:
 
-  * Δήλωση των στόχων
-  * Συγκεκριμένο προϊόν λογισμικού
-  * Συγκεκριμένο σχέδιο διοίκησης έργου
-  * Λίστα θεμάτων συνδεδεμένη με αυτό το προϊόν
-  * Διαδικασία τεχνικής επιθεώρησης
+  * Δήλωση των στόχων.
+  * Συγκεκριμένο προϊόν λογισμικού.
+  * Συγκεκριμένο σχέδιο διοίκησης έργου.
+  * Λίστα θεμάτων συνδεδεμένη με αυτό το προϊόν.
+  * Διαδικασία τεχνικής επιθεώρησης.
```

Comment:
For the sake of consistency (see below) and syntactical correctness, please separate with commas and start each line with a lowercase letter.